manim/camera/three_d_camera.py

```diff
@@ -163,15 +164,19 @@ def get_theta(self):
         """
         return self.theta_tracker.get_value()
 
+    @deprecated
```

Comment:
Don't we usually add some parameters to the deprecated decorator, like what something is replaced with and how long the method will still be available?